docs/lib/graphqlapi/fragments/android/authz/20_oidc.md

```diff
@@ -0,0 +1,80 @@
+Add the following code to your app:
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+// TODO
+public class MyOidcURLRequestInterceptor: URLRequestInterceptor {
```

Comment:
looking for some help here to fill in the java/kotlin's interceptor, @jpignata @raphkim ?

Reply:
@richardmcclellan @raphkim I have no idea haha. Can you weigh in here?

Reply:
Android doesn't yet allow custom interceptors, but we are looking to add that in soon. 
At the moment, the customer must implement `OidcAuthProvider` interface to return a token, and our interceptor factory will know how to generate an interceptor for that.

Reply:
is that something we want to advise the customer in the meantime or should we leave this section out of Android for now?

Reply:
i will make a commit to this branch with the instructions for adding OIDC auth provider to the category